githooks/build-indexes

```diff
@@ -312,13 +312,13 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 					}
 					banlist = Tools.getFormat(genPrefix + 'doublesuu').banlist;
 					if (!banlist) {
-						defaultTier = "DOU";
+						defaultTier = "DUU";
```

Comment:
This says "if there is no Doubles UU format in this gen, all pokemon are DUU" which is clearly wrong...

Reply:
i read this incorrectly then. fsr i read "!banlist" as "not on the banlist" instead of banlist existing, but it would have to like, check the pokemon first for that to be the case... so im dumb ig. When you have a moment can i talk you about this chunk of code to see if we can get it working as intended? see this post: http://www.smogon.com/forums/threads/bug-reports-v2-0-read-op-before-posting.3469932/page-373#post-7567416 if you haven't already

Reply:
Sure, find me on Discord or PS.